packages/uxpin-merge-cli/src/program/command/delete_version/getDeleteOptions.ts

```diff
@@ -0,0 +1,31 @@
+import { isDevelopmentEnv } from '../../../program/env/isDevelopmentEnv';
+import { isTestEnv } from '../../../program/env/isTestEnv';
+import { DeleteOptions } from '../../../steps/deleting/DeleteOptions';
+import { DeleteVersionArgs } from '../../args/ProgramArgs';
+import { getProjectRoot } from '../../args/providers/paths/getProjectRoot';
+import { getTempDirPath } from '../../args/providers/paths/getTempDirPath';
+
+const TEST_UXPIN_API_DOMAIN:string = '0.0.0.0';
+
+function getDefaultApiDomain(domain:string):string {
```

Comment:
this method exists in command/push/getBuildOptions.ts please move it to separate file

Reply:
similar to `RepositoryPointerType` enum is it good to move it out of both `getDeleteOptions` and `getBuildOptions` then import it from its own file?